assets/src/edit-story/components/helpCenter/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import { useFeatures } from 'flagged';
+import styled, { ThemeProvider } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { theme as dsTheme, ThemeGlobals } from '../../../design-system';
+import { Toggle } from './toggle';
+
+const Wrapper = styled.div`
+  position: absolute;
+  bottom: 16px;
+  left: 11px;
+  z-index: 10;
+
+  @media ${({ theme }) => theme.breakpoint.tablet} {
+    bottom: 24px;
+    left: 23px;
```

Comment:
Could we do 24px and 32px for line 41 left so that everything stays at the same spatial convention of the new designs? Sam has made note that sometimes the pixels in the specs are off a little and to just get as close as possible to - you can read more in the spacing section of the design system foundation